std/numeric.d

```diff
@@ -568,16 +568,28 @@ public:
     }
 
     /// ditto
+    // Define an opBinary `CustomFloat op CustomFloat` so that those below
```

Comment:
Since you are here, can you please also fix the type in `opBinary`s doc?
Line 556, from `relavent` to `relevant`.

Thanks

Reply:
Done.

Reply:
Nice. Thanks